modules/oe_whitelabel_extra_project/oe_whitelabel_extra_project.module

```diff
@@ -123,3 +124,26 @@ function _oe_whitelabel_extra_project_preprocess_inpage_nav(array &$variables):
   }
 
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function _oe_whitelabel_extra_project_preprocess_badges(&$variables) {
+  if ($variables['view_mode'] !== 'full' && $variables['view_mode'] !== 'teaser') {
```

Comment:
these checks have already been performed in oe_whitelabel_extra_project_preprocess_node__oe_project